app/forms/idv/phone_form.rb

```diff
@@ -25,7 +25,9 @@ def initialize(
       @user = user
       @allowed_countries = allowed_countries
       @delivery_methods = delivery_methods
-      self.phone = initial_phone_value(previous_params[:phone]) unless user_has_multiple_phones?
+      unless user_has_multiple_phones?
+        @international_code, @phone = initial_value(previous_params[:phone])
```

Comment:
(Previously we weren't setting `@international_code`, and the `<select>` rendered by `PhoneInputComponent` was just defaulting to `US`, since that was the first item in the list.)

Reply:
Is this a good opportunity to pull the `international_code` that's passed with `PhoneInputComponent` from `previous_params`, so that we don't have to try to infer the international code from the parsed phone number?

Reply:
Yeah that makes sense. We initially infer from the user's MFA phone, but then on submit we take whatever was in the form data.